src/admin-guide/environments/live-basic.yml

```diff
@@ -0,0 +1,8 @@
+name: cylc-8-live-basic
+# Cylc 8 without GUI installed from the default GitHub branch.
+channels:
+  - conda-forge
+dependencies:
+  - pip
```

Comment:
Should add a Python dependency (I think this is implicitly installing Python due to the pip dep)